### ISSUE TITLE
Does this approach work  with contenteditable and other components too?

### ISSUE THREAD

**Issue by sirajulm** [patrickmarabeas/nrg.js#1]
Hi, 
The demo looks good and performant, but i'd like to know if it will be possbile to work with contenteditable or other event handlers like click, from inside react so that the scope of the wrapper angular is updated appropriately. Thanks for the reply in advance.

**Comment by patrickmarabeas** [comment]
Hoo geez! This was just a thought experiment from waayyy back. Would suggest looking at [react2angular](https://github.com/coatue-oss/react2angular) or [ngReact]( https://github.com/ngReact/ngReact) instead. I've heard of both being used in production projects - and both are either more actively maintained / have a support network around them.

**Comment by sirajulm** [comment]
@patrickmarabeas I've gone throught react2angular, but it seemed impossible to pass on the updated value from React back to angular. Or atleast, my scope was not getting reflected immediate with the changes made in react. Since nrg.js was working as expected I'm proceeding with it. Let me know if there is any issues/cavets using the same. Also I've modifeid the code to work with contenteditable as well.

I'll go through react2angular and ngReact once again when I'll get time.

Thanks for the response.

**Comment by patrickmarabeas** [comment]
Yeah, seems like most libs stick with uni-directional data flow. Obviously depends on the context of your application as to what will suit best.

Just be weary that this was built with a 3+ year old version of React in mind. Uncertain how well the lifecycle methods will line up. I'm sure there will be better ways to mount the React components as well - I'd have to go dig into it all to remember how and why I did particular things.
